gcloud-java-bigquery/src/test/java/com/google/gcloud/bigquery/TableRowTest.java

```diff
@@ -0,0 +1,46 @@
+package com.google.gcloud.bigquery;
+
+import static org.junit.Assert.assertEquals;
+
+import com.google.common.collect.ImmutableList;
+
+import org.junit.Test;
+
+import java.util.List;
+
+public class TableRowTest {
+
+  private static final FieldValue BOOL_VALUE = new FieldValue(FieldValue.Kind.PRIMITIVE, "false");
+  private static final FieldValue INT_VALUE = new FieldValue(FieldValue.Kind.PRIMITIVE, "1");
+  private static final FieldValue FLOAT_VALUE = new FieldValue(FieldValue.Kind.PRIMITIVE, "1.5");
+  private static final FieldValue STRING_VALUE = new FieldValue(FieldValue.Kind.PRIMITIVE, "str");
+  private static final FieldValue TIME_VALUE = new FieldValue(FieldValue.Kind.PRIMITIVE, "42");
+  private static final List<FieldValue> ROW_VALUES =
+      ImmutableList.of(BOOL_VALUE, INT_VALUE, FLOAT_VALUE, STRING_VALUE, TIME_VALUE);
+  private static final TableRow TABLE_ROW1 = new TableRow.Builder()
```

Comment:
Could there theoretically be `FieldValue`s with value `null` in a TableRow?  If so I think we should add a `NULL_VALUE`